backend/test/Controllers/Admin/UserControllerTest.cs

```diff
@@ -139,47 +86,71 @@ public void GetAccessRequests_PageMax_Success()
+            var accessRequests = new[] { accessRequest1, accessRequest2 };
+            var paged = new Entity.Models.Paged<Entity.AccessRequest>(accessRequests);
+            service.Setup(m => m.User.GetAccessRequests(It.IsAny<int>(), It.IsAny<int>(), It.IsAny<string>(), It.IsAny<bool?>())).Returns(paged);
 
             // Act
             var result = controller.GetAccessRequests(2, 100);
 
             // Assert
             var actionResult = Assert.IsType<JsonResult>(result);
             var actualResult = Assert.IsType<Entity.Models.Paged<Model.AccessRequestModel>>(actionResult.Value);
-            Assert.Equal(mapper.Map<Model.AccessRequestModel[]>(expectedAccessRequests), actualResult.Items, new DeepPropertyCompare());
+            Assert.Equal(mapper.Map<Model.AccessRequestModel[]>(accessRequests), actualResult.Items, new DeepPropertyCompare());
             service.Verify(m => m.User.GetAccessRequests(2, 20, null, null), Times.Once());
         }
         #endregion
 
         #region GetUsers
         [Fact]
-        public void GetUsers()
+        public void GetUsers_Success()
         {
             // Arrange
-            var user = PrincipalHelper.CreateForPermission(Permissions.AdminUsers);
             var helper = new TestHelper();
-            var controller = helper.CreateController<UserController>(user);
+            var controller = helper.CreateController<UserController>(Permissions.AdminUsers);
 
             var mapper = helper.GetService<IMapper>();
             var service = helper.GetService<Mock<IPimsAdminService>>();
-            var expectedUsers = new Entity.User[] { _expectedUsers };
-            var expectedPagedUsers = new Entity.Models.Paged<Entity.User>(expectedUsers);
-            service.Setup(m => m.User.Get(It.IsAny<Entity.Models.UserFilter>())).Returns(expectedPagedUsers);
+            var users = new Entity.User[] { EntityHelper.CreateUser("user1"), EntityHelper.CreateUser("user2") };
+            var paged = new Entity.Models.Paged<Entity.User>(users);
+            service.Setup(m => m.User.Get(It.IsAny<Entity.Models.UserFilter>())).Returns(paged);
 
             // Act
             var result = controller.GetUsers();
 
             // Assert
             var actionResult = Assert.IsType<JsonResult>(result);
-            var actualAccessRequest = Assert.IsType<Entity.Models.Paged<Model.UserModel>>(actionResult.Value);
-            Assert.Equal(mapper.Map<Model.UserModel[]>(expectedUsers), actualAccessRequest.Items, new DeepPropertyCompare());
+            var actualResult = Assert.IsType<Entity.Models.Paged<Model.UserModel>>(actionResult.Value);
+            Assert.Equal(mapper.Map<Model.UserModel[]>(users), actualResult.Items, new DeepPropertyCompare());
             service.Verify(m => m.User.Get(It.IsAny<Entity.Models.UserFilter>()), Times.Once());
         }
         #endregion
 
+        #region GetUser
+        [Fact]
+        public void GetUser()
```

Comment:
It seems the pattern for tests are to name them `*_Success` but this one doesn't follow that naming...